components/frontend/src/changelog/ChangeLog.js

```diff
@@ -50,11 +51,14 @@ function ChangeLogWithoutMemo({ report_uuid, subject_uuid, metric_uuid, source_u
         if (source_uuid) {
             uuids.source_uuid = source_uuid
         }
-        get_changelog(nrChanges, uuids).then(function (json) {
-            if (!didCancel) {
-                setChanges(json.changelog || [])
-            }
-        })
+        get_changelog(nrChanges, uuids)
+            .then(function (json) {
+                if (!didCancel) {
+                    setChanges(json.changelog || [])
+                }
+                return null
+            })
+            .catch((error) => showMessage("error", "Could not fetch changes", `${error}`))
```

Comment:
Why not name it "list of changes" or "change log"?

Reply:
In the UI it says: "Changes in this report", "Changes to this subject", "Load more changes", etc.